docs/gh-pages-build.js

```diff
@@ -0,0 +1,142 @@
+#!/usr/local/bin/node
+/**
+ * Build a release for the gh-pages docs site.
+ */
+
+var fs = require('fs');
+var execSync = require('child_process').execSync;
+
+var usage = '\nbuild <vn.n.n[-pre[.n]] | HEAD> [-p]\n';
+var versionsFile = './src/www/versions.json';
```

Comment:
> Well, much of the git wrangling assumes we're in docs, and it would get super messy and hard-to-read if we path.resolve all of those.

I see, well you're right, I didn't take those into account :sweat_smile: 

> No harm in making it fool proof, but much easier to process.chdir('__dirname'); though I think?

That's actually a very smart move :+1: Yeah I think it's a lot safer :+1: